src/FluentHub/ViewModels/Users/DiscussionsViewModel.cs

```diff
@@ -53,6 +57,15 @@ private async Task RefreshDiscussionsAsync(string username)
             catch (Exception ex)
             {
                 _logger?.Error("RefreshDiscussionsAsync", ex);
+                if (_messenger != null)
+                {
+                    UserNotificationMessage notification = new("Something went wrong", ex.Message, UserNotificationType.Error);
+                    _messenger.Send(notification);
+                }
+                else
```

Comment:
Remove the `else`, throw the exception to the Discussions page so that it knows when to show the "refresh" button

Reply:
ok